easypost/easypost_object.py

```diff
@@ -1,86 +1,75 @@
 import json
-from typing import Any, Dict, List, Optional
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
 
 import easypost
 
 
+EASYPOST_OBJECT_ID_PREFIX_TO_CLASS_NAME_MAP = {
+    "adr": "Address",
+    "batch": "Batch",
+    "brd": "Brand",
+    "ca": "CarrierAccount",
+    "cstinfo": "CustomsInfo",
+    "cstitem": "CustomsItem",
+    "evt": "Event",
+    "hook": "Webhook",
+    "ins": "Insurance",
+    "order": "Order",
+    "pickup": "Pickup",
+    "pickuprate": "PickupRate",
+    "pl": "PostageLabel",
+    "plrep": "Report",
+    "prcl": "Parcel",
+    "rate": "Rate",
+    "refrep": "Report",
+    "rfnd": "Refund",
+    "sf": "ScanForm",
+    "shp": "Shipment",
+    "shpinvrep": "Report",
+    "shprep": "Report",
+    "trk": "Tracker",
+    "trkrep": "Report",
+    "user": "User",
```

Comment:
`webhook` is missing from the list. We may want to check it to ensure nothing else is missing.

Reply:
It's "hook"

Reply:
Ah thanks, skipped right over it ha.